Jenkinsfile

```diff
@@ -873,6 +944,14 @@ pipeline {
             }
         }
         stage('Unit Test') {
+            when {
+                beforeAgent true
+                // expression { skipTest != true }
+                expression {
+                    sh script: 'git show -s --format=%B | grep "^Skip-test: true"',
```

Comment:
Staging needs to skip tests also.
                    if [ "${NO_CI_TESTING}" == 'true' ]; then ...

Reply:
But isn't skipping tests on staging already working?

Reply:
Skipping tests on staging is skipping implemented in the runTest script.

So if this change only affects tests run in containers, than nothing special for changing is needed.

Reply:
I don't think whether what's being skipped is in a container is relevant.  I think the overall point is that these changes shouldn't be changing anything in the way staging's skipping is working.  I think the [run results on staging](https://build-stage.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-879/3/pipeline) confirm that.